README.md

```diff
@@ -12,19 +12,40 @@ This is the official Go client library for the [IPinfo.io](https://ipinfo.io) IP
 
 Check all the data we have for your IP address [here](https://ipinfo.io/what-is-my-ip).
 
-### Getting Started
+
+- [Getting Started](#getting-started)
+	- [Installation](#installation)
+	- [Quickstart](#quickstart)
+- [Authentication](#authentication)
+- [Internationalization](#internationalization)
```

Comment:
Can we also add `CountryFlag` here that was merged in the last commit?

Reply:
Hey, @rm-Umar, what would be the best way to go about this? Shall we merge this issue now, then I can make a second PR specifically for the `CoutnryFlag` section?